// error handling -> try, catch, finally,throw
// try..... catch handle run time errors(program which is )
//Error object
//finally statement -> execute code, after try and catch

try {
  //code test
  alert("hi everyone");
  alert(x);
  alert("im abida");
  alert("bye everyone");
} catch (err) {
  // console.log(err);
  alert("inside catch block");
} finally {
  alert("bye everyone");
}
